wordle-web/tests/wordList.test.ts

```diff
@@ -0,0 +1,142 @@
+import { describe, it, test, expect } from 'vitest';
+import { Game } from '~/scripts/game';
+import { myWordList } from '~/scripts/wordList';
+import { ValidWordsUtils } from '~/scripts/validWordsUtils';
+
+const wordList = [
+  'AARGH',
+  'ABACA',
+  'HELLO',
+  'THINK',
+  'PEACH',
+  'BEACH',
+  'WHICH',
+];
+
+const wordList2 = ['ABACA', 'ABODE', 'TAROT'];
+
+describe('validWords_enteredLettersIn_returnsCorrectWordAndLength', () => {
+  it('enteredA_returnsFirstTwoWords', () => {
+    let game = new Game();
+    game.secretWord = 'HELLO';
+    game.addLetter('A');
+    let utils = new ValidWordsUtils(game, wordList);
+    let validWords = utils.validWords();
+    expect(validWords.length).toBe(2);
+    expect(validWords[0]).toBe('AARGH');
+    expect(validWords[1]).toBe('ABACA');
+  });
+  it('enteredAA_returnsAARGH', () => {
+    let game = new Game();
+    game.secretWord = 'HELLO';
+    game.addLetter('A');
+    game.addLetter('A');
+    let utils = new ValidWordsUtils(game, wordList);
+    let validWords = utils.validWords();
+    expect(validWords.length).toBe(1);
+    expect(validWords[0]).toBe('AARGH');
+  });
+  it('noLettersEntered_returnsFullList', () => {
+    let game = new Game();
+    game.secretWord = 'HELLO';
+    let utils = new ValidWordsUtils(game, wordList);
+    let validWords = utils.validWords();
+    expect(validWords.length).toBe(7);
```

Comment:
```suggestion
    expect(validWords.length).toBe(wordList.length);
```
Hardcoding 7 works fine because we know the length of the list at time of writing, however we can guard against future changes here in addition to making the assert portion read a bit clearer: "expect validWords to have the same count as the wordList"